core-plugins/docs/HDFSAction-action.md

```diff
@@ -0,0 +1,45 @@
+# HDFS Action
+
+
+Description
+-----------
+Moves file(s) within HDFS.
+
+
+Use Case
+--------
+This action can be used when files needs to be moved to a new location in HDFS for the purpose of archiving,
```

Comment:
perhaps? Ericsson just wants to be able to move files in HDFS, so that's why i just went with the `rename()` call`